client/me/concierge/cancel/index.js

```diff
@@ -58,7 +58,9 @@ class ConciergeCancel extends Component {
 							href={ `/me/concierge/${ siteSlug }/book` }
 							primary={ true }
 						>
-							{ translate( 'Schedule' ) }
+							{ translate( 'Schedule', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'Schedule…' )` *ES Score: 17*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/i18n-schedule-concierge-session)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `customizer changeset action/button label`
- `post action/button label`

 Would it make sense to reuse one of the above?